bescort.lic

```diff
@@ -801,7 +801,7 @@ class Bescort
   def power_walk(pattern)
     fput 'exit' if dead? || Flags['ap-danger']
     if Flags['harness-check']
-      harness_mana(20) unless Flags['harness-check'].include?('effortlessly')
+      harness_mana(['20']) unless Flags['harness-check'].include?('effortlessly')
```

Comment:
Why is this now a string?

Reply:
It's an array?

Reply:
Seemed better to make harness_mana use an array rather than check if there was an array coming in and doing something different for it.

Reply:
Am I doing it wrong?

Reply:
Works for my testing?

Reply:
An Array can contain integers. If we really want a number, then I think it should be a number. This happens to work because all we do is some string formatting, but would break if we did any math later (see discern logic).

Reply:
There is a separate conversation to be had around whether or not this should take an array. Specifically, I'm a bit leery around changing the signature of existing `common-*` methods since they can cause problems for people running multiple characters.

Reply:
Yeah, but it seems pointless to have to check for an array when we can just use it, seems better to just say "restart dependency' than to put it in in a clunky way.

Reply:
I see what you mean by number/string, my mistake, duh.